### ISSUE TITLE
abnormal beige adipocyte physiology

### ISSUE THREAD

**Issue by sbello** [obophenotype/mammalian-phenotype-ontology#2246]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
abnormal beige adipocyte physiology

>2. Synonyms (e.g., absent spleen)
abnormal beige adipocyte physiology | abnormal beige brown adipocyte physiology | abnormal brite adipocyte physiology | abnormal brite cell physiology | abnormal brite fat cell physiology 

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
any functional anomaly of fat cells that are beige in color, thermogenic, and which differentiates in white fat tissue from a Myf5-negative progenitor (from MP:0020325)
J:240649

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

MP:0005666

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

**Comment by sbello** [comment]
MP:0020426
